src/components/Cards/Cards.jsx

```diff
@@ -4,15 +4,23 @@ import { Row, Col, Badge } from "reactstrap";
 import { GatsbyImage } from "gatsby-plugin-image";
 
 const Cards = ({ title, cover, tags, excerpt, path }) => {
+  let tagCount = tags.length - 2;
   return (
     <Col className="ml-4 mr-4 mt-2 mb-2 " lg="3" md="8" sm="12">
       <Row className="row_shadow h-100">
         <Col lg="12" md="12" className="p-0">
           <div className="card-img search-cards">
             <GatsbyImage image={cover.gatsbyImageData} />
           </div>
-          <div className="md-tag mt-3 ml-3">
-            <Badge>{tags}</Badge>
+          <div className="d-flex">
+            <div className="md-tag mt-3 ml-3">
+              {tags.slice(0, 2).map((tag) => {
+                return <Badge className="mr-2">{tag}</Badge>;
+              })}
```

Comment:
I believe you don't need the return if I remember my JSX correctly.

Reply:
We can directly do this thing removing curly brackets and return , but we are passing html elements so returning them looks clear and readable,

Reply:
```suggestion
              {tags.slice(0, 2).map((tag) => <Badge className="mr-2">{tag}</Badge>)}
```